CRS-JAVA-PROJECT-IMP/src/main/java/com/flipkart/client/LoginSignupDashboard.java

```diff
@@ -29,13 +29,13 @@ public static User LoginUser() throws CRSException {
 
             switch (roleOfUser){
                 case ADMIN:
-                    // TODO : complete here
+                    new AdminDashboard(user).helper();
```

Comment:
It would be better if you can create Admin object instead of User so that admin we can access admin fields as well. and same goes with Student and Professor